src/Microsoft.Health.Fhir.Core/Logging/Metrics/CrudMetricHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Logging.Metrics
+{
+    public sealed class CrudMetricHandler
```

Comment:
Sorry to back track here, but I don't know if it makes sense to have both a handler and an empty IFhirMetricEmitter, I think we only should have of those, no need for both. I like the filter approach though, encapsulates the logic nicely.